examples/foundational/07ab-interruptible-inworld-http.py

```diff
@@ -58,45 +87,43 @@
 
 
 async def run_bot(transport: BaseTransport, runner_args: RunnerArguments):
-    logger.info(f"Starting bot")
+    logger.info("Starting bot")
 
-    # Create an HTTP session
     async with aiohttp.ClientSession() as session:
         stt = DeepgramSTTService(api_key=os.getenv("DEEPGRAM_API_KEY"))
 
-        # Inworld TTS Service - Unified streaming and non-streaming
-        # Set streaming=True for real-time audio, streaming=False for complete audio generation
-        streaming = True  # Toggle this to switch between modes
-
-        tts = InworldTTSService(
+        tts = LoggingInworldTTSService(
             api_key=os.getenv("INWORLD_API_KEY", ""),
             aiohttp_session=session,
             voice_id="Ashley",
             model="inworld-tts-1",
-            streaming=streaming,  # True: real-time chunks, False: complete audio then playback
+            params=InworldTTSService.InputParams(timestamp_type="WORD"),
         )
 
         llm = OpenAILLMService(api_key=os.getenv("OPENAI_API_KEY"))
 
         messages = [
             {
                 "role": "system",
-                "content": "You are very knowledgable about dogs. Your output will be spoken aloud, so avoid special characters that can't easily be spoken, such as emojis or bullet points. Respond to what the user said in a creative and helpful way.",
+                "content": "You are a helpful AI demonstrating Inworld AI's TTS. Your output will be spoken aloud, so avoid special characters that can't easily be spoken, such as emojis or bullet points. Respond to what the user said in a friendly and helpful way.",
             },
         ]
 
         context = LLMContext(messages)
         context_aggregator = LLMContextAggregatorPair(context)
 
+        rtvi = RTVIProcessor(config=RTVIConfig(config=[]))
```

Comment:
This can be simplified.

```suggestion
        rtvi = RTVIProcessor()
```